src/clib/grackle_chemistry_data.h

```diff
@@ -136,6 +136,21 @@ typedef struct
   /* flag for Wolcott-Green+ 2011 H2 self-shielding */
   int H2_self_shielding;
 
+  /* flag to select which formula for calculating k11 you want to use. Setting to 1 will use Savin 2004, 0 will use Abel et al. 1997  */
+  int h2_charge_exchange_rate;
+
+  /* flag to select which formula for calculating h2dust you want to use. Setting to 1 will use Omukai 2000, 2 will use Hollenbach & McKee (1979) */ 
+  int h2dust_rate;
+
+  /* flag to select which formula for calculating low density H2 cooling rate due to H collisions. Setting to 1 will use Lique 2015 */
```

Comment:
Is "1" the only option here?

Reply:
1 uses Lique 2015, 2  uses Glover & Abel 2008. I have added this clarification here as well as updating the rate_function with the additional comments found in the Fortran for verbosity.